src/System.Text.Json/src/System/Text/Json/JsonHelpers.cs

```diff
@@ -24,39 +24,39 @@ public static bool IsValidUnicodeScalar(uint value)
         }
 
         /// <summary>
-        /// Returns <see langword="true"/> iff <paramref name="value"/> is between
+        /// Returns <see langword="true"/> if <paramref name="value"/> is between
```

Comment:
Was iff originally written to mean if and only if?

Reply:
Maybe? But that has no meaning and isn't consistent with normal commenting.

Reply:
Yes, iff was meant to be shorthand for if and only if.

This is used all over the place.
https://github.com/dotnet/corefx/search?q=iff&unscoped_q=iff

I don't think we should change `iff` to `if`, especially since most of these are internal utility methods, so clear/unambiguous docs don't hurt.

However, if you think we should remove it, there are a couple places you missed:
https://github.com/dotnet/corefx/blob/81b393ccc67dd8126996ab7f652f2a7e05726516/src/System.Text.Json/src/System/Text/Json/Writer/JsonWriterHelper.Escaping.cs#L176-L184

Reply:
Got the others.

Reply:
It is a word (according to Websters)
```
iff conjunction
\ ˈif-ᵊn(d)-ˈōn-lē-ˌif  ; ˈif, sometimes read with a prolonged f  \
Definition of iff (Entry 1 of 2)
: if and only if
two figures are congruent iff one can be placed over the other so that they coincide
—used in logic and mathematics
```

Reply:
> It is a word (according to Websters)

My preference is to keep using it where it's being used, but I don't feel strongly.

It existed in the original location that I borrowed these utility methods from.

Reply:
I know it is a word, I meant to say it has no additional value over "if". It just makes it harder to read and leaves you wondering why "iff" was used if you happen to know "iff" wasn't a typo.

Reply:
> I meant to say it has no additional value over "if".

If you just state "if", then what does the method return when the value is not within the range (i.e what happens when the pre-condition for returning true isn't met)? In that case, the method could return either true or false since the "else" case isn't specified (and hence it's ambiguous). Although, you can usually infer that it would return false and the "otherwise" clause is implied. So, there is **some** value in using "iff".

The comment with "iff":
```text
Returns <see langword="true"/> iff <paramref name="value"/> is a valid Unicode scalar value, 
i.e., is in [ U+0000..U+D7FF ], inclusive; or [ U+E000..U+10FFFF ], inclusive.
```

Is equivalent to the following but is less verbose (since the else condition for returning false doesn't need to be specified):
```text
Returns <see langword="true"/> if <paramref name="value"/> is a valid Unicode scalar value, 
i.e., is in [ U+0000..U+D7FF ], inclusive; or [ U+E000..U+10FFFF ], inclusive.
Otherwise, returns <see langword="false"/>.
```

> It just makes it harder to read and leaves you wondering why "iff" was used if you happen to know "iff" wasn't a typo.

That's definitely a fair point. If we are **nitpicking** here, for clarity, maybe add the "otherwise" clause then, when removing "iff" :)

Nonetheless, **definitely** not worth changing/resetting CI over, especially since its just personal preference and the comment is only visible internally.